xtask/src/cli.rs

```diff
@@ -117,24 +117,35 @@ impl Subcommand {
 }
 
 pub fn run(command: &mut Command) -> Result<()> {
-    println!("Running: {}", format_command(command));
+    let command_str = format_command(command);
+    println!("Running: {}", command_str);
 
     let output = command
         .stdout(Stdio::piped())
```

Comment:
Oh, I missed that we changed it to capture output. @mejrs, @messense how do you feel if we change this so that we just inherit output handles instead? I think it's nice to see what's happening in realtime; `nox` for example doesn't squash output by default.

Reply:
> how do you feel if we change this so that we just inherit output handles instead?

That's also fine.

Reply:
>  how do you feel if we change this so that we just inherit output handles instead?

I wasn't really thinking that much when I wrote this, so a better approach would be welcome 👍